### ISSUE TITLE
Why is output same regardless of epochs and learning_rate

### ISSUE THREAD

**Issue by za13** [Wslsdx/Deep-Learning-Approach-for-Surface-Defect-Detection#14]
I tried varying the number of epochs and learning rate, but I get pretty much the exact same output for the predicted defects no matter what. I tried epochs ranging from 10 to 500 or so, and varying the learning rate from 0.1 to 1e-9. Doesn't make sense to me

**Comment by fighting666777** [comment]
I have the same problem，did you solve it?

**Comment by fighting666777** [comment]
> I tried varying the number of epochs and learning rate, but I get pretty much the exact same output for the predicted defects no matter what. I tried epochs ranging from 10 to 500 or so, and varying the learning rate from 0.1 to 1e-9. Doesn't make sense to me

I have the same problem，did you solve it?
